augly/tests/image_tests/transforms_unit_test.py

```diff
@@ -25,6 +25,9 @@ def test_Blur(self):
 
     def test_Brightness(self):
         self.evaluate_class(imaugs.Brightness(), fname="brightness")
+    
+    def test_Skew(self):
```

Comment:
please insert this function in alphabetical order with respect to the other functions in this file

Reply:
Done !